Directory.Build.props

```diff
@@ -13,5 +13,15 @@
     <TargetsWindows Condition="$(RuntimeIdentifier.StartsWith('win'))">true</TargetsWindows>
     <TargetsLinux Condition="$(RuntimeIdentifier.StartsWith('linux'))">true</TargetsLinux>
     <LangVersion>Latest</LangVersion>
+    <!-- We use the compiler toolset that comes from NuGet Packages rather than the SDK built-in.
+    This one sets UseSharedCompilation to false by default. -->
+    <UseSharedCompilation>true</UseSharedCompilation>
+
+    <CompilerVersion>3.0.0-beta3-final</CompilerVersion>
   </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Condition="'$(MSBuildRuntimeType)' != 'Core'" Include="Microsoft.Net.Compilers" Version="$(CompilerVersion)" IsImplicitlyDefined="true" />
```

Comment:
What does the IsImplicitlyDefined do?

Reply:
It is for the IDE so that people can't remove the package reference from VisualStudio UI.